<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QDialog" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:rgb(0, 0, 0);
color:white
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="continueBtn">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:rgb(255, 255, 255);
background-color:rgb(0, 0, 0);
border: 2px solid rgb(124, 124, 124);
solid: rgb(255, 255, 255);
border-radius:15px;
max-height:30px;
max-width:80px;
min-height:30px;
min-width:80px;
}</string>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="regBtn">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:rgb(255, 255, 255);
background-color:rgb(0, 0, 0);
border: 2px solid rgb(124, 124, 124);
solid: rgb(255, 255, 255);
border-radius:15px;
max-height:30px;
max-width:1200px;
min-height:30px;
min-width:80px;
}</string>
      </property>
      <property name="text">
       <string>registration</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="title">
    <string>Registration</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 14pt &quot;Segoe UI&quot;;
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string> Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color:rgb(255, 255, 255);
background-color:rgb(0, 0, 0);
border: 2px solid rgb(124, 124, 124);
solid: rgb(255, 255, 255);
border-radius:15px;
max-height:30px;
max-width:280px;
min-height:30px;
min-width:80px;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Normal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 14pt &quot;Segoe UI&quot;;
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string> Login      </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color:rgb(255, 255, 255);
background-color:rgb(0, 0, 0);
border: 2px solid rgb(124, 124, 124);
solid: rgb(255, 255, 255);
border-radius:15px;
max-height:30px;
max-width:300px;
min-height:30px;
min-width:80px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 14pt &quot;Segoe UI&quot;;
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string> Name     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color:rgb(255, 255, 255);
background-color:rgb(0, 0, 0);
border: 2px solid rgb(124, 124, 124);
solid: rgb(255, 255, 255);
border-radius:15px;
max-height:30px;
max-width:300px;
min-height:30px;
min-width:80px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
